countme-trim-raw.py

```diff
@@ -207,7 +208,7 @@ def _main():
 
         connection = None
         while True:
-            connection, week = get_trim_data(args.sqlite, connection=connection)
+            connection, week = get_trim_data(args.sqlite, connection=connection, before=before)
```

Comment:
We are still running this in a loop that deletes weaks, so the report is now wrong. We also probably don't ever want to delete partial weeks of countme data, which is what happens if we go further and remove the loop completely (because a partial week of data is not usable, and can only corrupt totals.db).
Maybe you want to just print something to make it obvious that it'll delete a week eight times when given 8w as a duration.